values/cert-manager/cert-manager-raw.gotmpl

```diff
@@ -66,13 +72,20 @@ resources:
+      tls.key: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBeGlQS0lVOWt5WFlvak8vNVRqaTJPZGdGcXdyRnBhL2hkVEQ4MjhUaHg3V1h3M0M5CkcyNml6YXFhMVk5aFJVUWNlUHRwY2lGdnJHMUQyNUdSM2tVL3FBYjgwbkZHYXQzcDlTRVdOdXFYTGZram5pSksKRVYzY1kwcGZqSWZlMHU1S0dBSWlVaElUQi9CazRoaEovWHpzNjkrbDArMGpPd2plakNTU0lmOWY2RmF6d2dGQQppYUJ6TTViY0xLWk5XbmVrQlFhT1NJbDgvRG1ua0pGNkJXdHJubTJFYVNmbWR2T25RYklkbkI3OXk2UmVBV0FOCjZBc20zVVhNUkFPM1V3R3lGSkJEbWZiRklVek1PVGl3QkxJdjEwbGJ1VHZJOU9MRkM4aWlKdzVRY093RVZiUEEKQ21FQ09hbThFdkNvUHQ4bW1mVE0xbjhibUFtWGhXc1RxTE9jWFFJREFRQUJBb0lCQUhzUHYzcncxS2hMdXVQTwpLQWx0TDhDOWFzZlE2UEQvVEFoOVpJOGF1U0JRcVBHNkU3bjEvU2Uvem1yVm9wUml1VXhxMmhNSXNWTm1pYjdWCnFzTmlQeWNkTCttRWNUZnVuR0NqTENqdml2WUN4eHh3TDVDWnUwZnY2V2hlK3dZRytmM3ZWV0dqMkM0V0lKSk0KZzZmczd6bVFybTN4MnUvUGttbDRUVXJRVWNEN0FMZytVTkt3dTRLdnJ6UUxmWVd2VHpxdDl3ZzBlQlhicHkwTgpMbGJ4bFZTSklMbWRzei9xQTB5ZHZyc05nSTVSMlFOS0VVWHJMZklNYlM5ZmdIb2VTamI2ZmhUSGo1YWE2S0N0Cmd3ZU5tT2hmb0tjZk9PSW13c3lleVJuemZid0FXRzBLd1orWkNiOHR6RHZwbGQwSGRwOTVISmdmaXlOZDk4ZEgKSkRSb3dBRUNnWUVBNlhydWVkTnZxblF3OWVYR1Q3WHA5b0U5OEFySmZ1ODFPOEM4WWZHcTNQWmM5WEF0T0Y0eQo3SFRnaFhINlFsOE9HMTRTKytuU2kzSGdzOW5mQk5SYXNreEl0Mk9iY0ZRU1BJTzgzSGNxc3ZtY2txcmh3OUFxCmM0WmFSN3FxTkdaRmlkT2VZQ21VcmF6Y3hqQk51SStqdFhsNk41aWRuWGM0R2VFS2FpVXkvN0VDZ1lFQTJVQTgKbExGWndGKzY2MGpyd1NoTDc2cHhYTEZtb2M5TzU3dGdtRUNpRnhJakpFbmxFSWJiclpOMEk2NStoVS9rZ09VWAp6eE1zbTZOaFJycjRIcHcrdUdjK1c5QjhHOXR6dVRJZjFBWXQ3TlFQdjFwdGlzVWY3Mk1CaVpBOGc1ZUsrcUFOCkI5MENBZUhJM1JITGVvTEZZWllkb0dkZFB1bzRYVFRqTk0vbUhtMENnWUE5NjhsMWE5cndwUEliTlA0OFE2MnkKSDZLeWw5bktuVnhMNVpqUWpReFluOG9WbWRPVVlIODI4T1JlMzlTVVo2dFJITWtDSUlnalRXZGZxNGVYTzA1MgpmUDdJREZXOEc3L2VacjNlNW9mZmdZM0I1Q29PQ1paSW56Z3dUOVEySTIyTDJTRG10Z0JsY3pQMkovZGszRXQ1CnBLY0NPYXFMaVlQS2NVR1cxeGpJY1FLQmdRRE5CTkRtNWNNMFIyTW5YVWFXbjBlaUlqQWU4STlSVGtsQmFOUU0KZXBvOHVYSU9jZW5Jd2V1WUQ1by9maEdLRDA0dDBZUzVkM2ZXWE1XVk1TYXNCSGt3SHNaN2NQRmVmS2ZsOG9hVQoyVXdIdmt3cUlkVlVheGpyR1h5KzdDNGZaU2pUd2F6cWFjZldMTlFZeEExSlRSV21OVHM2UDdHQWlScXQ0eFl6CnBhcWR0UUtCZ1FDTnZjdExueVhhcWRaWnc0SGxoM21RQzVCbHEyaFdQaFE1REI3bEFJS3NURXdEZmNZTHQzK2IKd2UxekJqelloUTUwTEdrZUhkcWlLQjM4enJFNmJ1QTBBY1BOcGhDNXBQcVVXRzdFc2RheGdwV3BhRUpZc3Y1Vwp4V0hZY2FXVitUcy8wcmdBVEliTXlzSkVKK2VTRU93NHRUdmxXNHdNc2o2dHhtL0JoeXRnZlE9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
I think if we build a mechanism to auto rotate it would be a bit overkill/overengineered for now, let's first see how it goes with everyone using this integration test?

Reply:
This shall be provided from values IMO.
The certificate shall be required if a user selects `cert-mnager.stage: selfsigned-issuer`